### PR TITLE
[fm-equalize] Add DumpFMEParams script

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds DumpFMEParams script.

Draft: #13659 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/fm-equalize/fmelib/ComputeParam.py [hunk 1]
```diff
@@ -37,3 +37,13 @@ def _channelwiseMinMax(tensors: Tensors, channel: int) -> Tuple[List[float], Lis
         channel_wise_min.append(float(np.min(min_act[:, :, :, c])))
         channel_wise_max.append(float(np.max(max_act[:, :, :, c])))
     return channel_wise_min, channel_wise_max
+
+
+def getActivationMax(tensor: np.ndarray) -> float:
+    """
+    Get max values of activation.
+    :param tensors: a list of numpy array (each is a tensor)
+    :return: a list of scale (float) for each channel
```
Comment (jinevening) [inline compiler/fm-equalize/fmelib/ComputeParam.py:46]: Maybe?
```suggestion
    :return: a list of max (float) of absolute values for each channel
```

Comment (mhs4670go) [reply compiler/fm-equalize/fmelib/ComputeParam.py:46]: That sounds more better. I just followed "activation scale" in the smooth quant [codes](https://github.com/mit-han-lab/smoothquant/blob/c61476d728e42ae0d8a35e7e78494edcac3237b5/smoothquant/smooth.py#L19).

File: compiler/fm-equalize/fmelib/ComputeParam.py [hunk 2]
```diff
@@ -37,3 +37,13 @@ def _channelwiseMinMax(tensors: Tensors, channel: int) -> Tuple[List[float], Lis
         channel_wise_min.append(float(np.min(min_act[:, :, :, c])))
         channel_wise_max.append(float(np.max(max_act[:, :, :, c])))
     return channel_wise_min, channel_wise_max
+
+
+def getActivationMax(tensor: np.ndarray) -> float:
```
Comment (jinevening) [inline compiler/fm-equalize/fmelib/ComputeParam.py:42]:
```suggestion
def getActivationMax(tensor: np.ndarray) -> np.ndarray:
```

Comment (jinevening) [reply compiler/fm-equalize/fmelib/ComputeParam.py:42]: Is this right?

File: compiler/fm-equalize/fmelib/ComputeParam.py [hunk 3]
```diff
@@ -37,3 +37,13 @@ def _channelwiseMinMax(tensors: Tensors, channel: int) -> Tuple[List[float], Lis
...
+    """
+    Get max values of activation.
+    :param tensors: a list of numpy array (each is a tensor)
+    :return: a list of max (float) of absolute values for each channel
```
Comment (jinevening) [inline compiler/fm-equalize/fmelib/ComputeParam.py:46]:
```suggestion
    :return: 1D array of max of absolute values for each channel
```

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**